### PR TITLE
Feat/services tabs

### PR DESCRIPTION
**Pull request by marie-j**
## Move sidebar item into tabs 

⚠️ PR looks big but most of it is moving files ⚠️ 

### Description of the Change

* Revamp existing routing
* Replace menu with header tabs

MANAGER-3045

### CODE REVIEW

File: client/app/billing/autoRenew/billing-autoRenew.controller.js [hunk 1]
```diff
@@ -70,6 +78,10 @@ angular.module('Billing.controllers').controller('Billing.controllers.AutoRenew'
       text: $translate.instant('autorenew_service_type_ALL'),
     };
 
+    $scope.agreementsLink = agreementsLink;
+    $scope.homeLink = homeLink;
+    $scope.currentActiveLink = () => currentActiveLink();
+    $scope.isEnterpriseCustomer = isEnterpriseCustomer;
```
Comment (marie-j) [inline client/app/billing/autoRenew/billing-autoRenew.controller.js]: Went through `$scope` and did not turn into a component as revamp is planned

File: client/app/billing/autoRenew/ssh/user-ssh.html [hunk 2]
```diff
@@ -39,14 +33,14 @@ <h1 data-translate="user_ssh_title"></h1>
             </button>
             <oui-dropdown-content>
                 <button class="oui-dropdown-option"
-                        data-ng-click="setAction('ssh/add/dedicated/user-ssh-add-dedicated', ctrlSsh.sshKeyList)"
+                        data-ng-click="setAction('ssh/add/dedicated/user-ssh-add-dedicated', ctrlSsh.sshKeyList, '', true)"
                         data-track-name="account::ssh::add::dedicated::add_ssh_key"
```
Comment (euhmeuh) [inline client/app/billing/autoRenew/ssh/user-ssh.html]: We'll need to find a way to get rid of those `setAction` calls in favor of actual state transitions.

File: client/app/billing/payment/ovhAccount/billing-payment-ovhAccount.routes.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ angular
 
         $stateProvider.state(name, {
           url: '/ovhaccount',
-          templateUrl: 'account/billing/payment/ovhAccount/billing-ovhAccount.html',
+          templateUrl: 'billing/payment/ovhbilling-ovhAccount.html',
```
Comment (JeremyDec) [inline client/app/billing/payment/ovhAccount/billing-payment-ovhAccount.routes.js]:
```suggestion
          templateUrl: 'billing/payment/ovhBilling-ovhAccount.html',
```

File: client/app/billing/billing.routing.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+      denyEnterprise: ($q, currentUser) => {
+        if (currentUser.isEnterprise) {
+          return $q.reject({
+            status: 403,
+            message: 'Access forbidden for enterprise accounts',
+            code: 'FORBIDDEN_BILLING_ACCESS',
+          });
+        }
+
+        return true;
```
Comment (JeremyDec) [inline client/app/billing/billing.routing.js]: Shouldn't it be `false` ? If `current.isEnterprise` === false, then we don't deny entreprise ?

File: client/app/billing/payment/ovhAccount/billing-payment-ovhAccount.routes.js (hunk 3 above)
Comment (marie-j) [reply client/app/billing/payment/ovhAccount/billing-payment-ovhAccount.routes.js]: Nice catch

File: client/app/billing/autoRenew/agreements/user-agreements.html [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+    <div class="alert"
+            data-ng-class="alertType"
+            data-ng-if="message">
+        <button type="button"
+                class="close"
+                data-dismiss="alert"
+                aria-label="Close"
+                data-ng-click="resetMessages()"
+                title="{{ 'user_agreements_alert_close' | translate }}">
```
[the remark this answers is not in the document]
Comment (marie-j) [reply client/app/billing/autoRenew/agreements/user-agreements.html]: Only moved those files around, can be taken care of in another revamp

File: client/app/billing/autoRenew/billing-autoRenew.html [hunk 6]
```diff
@@ -0,0 +1,341 @@
...
+            </div>
+
+            <form class="row"
+                  data-ng-if="nicBillingFilter.values.length > 1">
+                <div class="form-group col-md-4">
+                    <label for="billingAutoRenewNicBillingSelect"
+                           class="control-label"
+                           data-translate="autorenew_service_autorenew_contact_filter">
+                    </label>
+                    <select id="billingAutoRenewNicBillingSelect"
```
[the remark this answers is not in the document]
Comment (marie-j) [reply client/app/billing/autoRenew/billing-autoRenew.html]: Will be revamped soon enough :)

File: client/app/billing/autoRenew/ssh/user-ssh.controller.js [hunk 7]
```diff
@@ -7,7 +7,16 @@ angular.module('UserAccount').controller('UserAccount.controllers.ssh', [
...
-  function ($scope, $q, $translate, UseraccountSshService, User, constants, $log, Alerter) {
+  function (
+    $scope,
+    $q,
+    $translate,
+    UseraccountSshService,
+    User,
+    constants,
+    $log,
+    Alerter,
```
[the remark this answers is not in the document]
Comment (marie-j) [reply client/app/billing/autoRenew/ssh/user-ssh.controller.js]: MANAGER-3062

File: client/app/billing/autoRenew/translations/Messages_fr_FR.json [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+  "autorenew_service_action_menu_smsPack_credit": "Ajouter des crédits",
+  "autorenew_service_action_menu_smsPack_automaticRenew": "Configurer la recharge automatique",
+  "autorenew_service_autorenew_active_checkbox_label": "Autoriser le renouvellement automatique sur les services désignés",
+  "autorenew_service_autorenew_day": "Jour de renouvellement automatique:",
+  "autorenew_service_autorenew_day_tooltip": "Si vous avez des services en renouvellement automatique, vous pouvez choisir leur jour de renouvellement",
+  "autorenew_service_autorenew_day_inactive": "Inactif",
+  "autorenew_service_autorenew_day_inactive_warning": "Les renouvellements automatiques ne fonctionnent que si un jour de renouvellement est sélectionné.",
+  "autorenew_service_autorenew_guide_help": "Besoin d'aide pour vos renouvellements ?",
+  "autorenew_service_autorenew_update": "Mise à jour de vos paramètres de renouvellement automatique.",
+  "autorenew_service_update_renewal_activation_notice": "Le renouvellement automatique a été activé sur le(s) service(s) désigné(s) ainsi que votre compte.",
```
[the remark this answers is not in the document]
Comment (marie-j) [reply client/app/billing/autoRenew/translations/Messages_fr_FR.json]: Will revamp translations, just extracted them for now

File: client/app/billing/billing.routing.js (hunk 4 above)
Comment (marie-j) [reply client/app/billing/billing.routing.js]: I suppose this was to differentiate the Promise rejection from else but sounds more logical indeed :)

File: client/app/billing/billing.module.js [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    swsProxyRootPath: config.swsProxyRootPath,
+    paymentMeans: ['bankAccount', 'paypal', 'creditCard', 'deferredPaymentAccount'],
+    target: config.target,
+  })
+  .constant('LANGUAGES', config.constants.LANGUAGES)
+  .constant('Billing.URLS', {
+    renew: config.constants.billingRenew,
+  })
+  .config(routing)
+  .run(($rootScope, coreConfig) => {
```
Comment (antleblanc) [inline client/app/billing/billing.module.js]: Not yours but:
```suggestion
  .run(/* @ngInject */ ($rootScope, coreConfig) => {
```

Comment (marie-j) [reply client/app/billing/billing.module.js]: @antleblanc I've added it but will pop in another PR

**Pull request merged**